impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Radar.js

```diff
@@ -183,6 +189,14 @@ define([
           rowIndexes: rowIndexes
         };
 
+        var tooltipString = record.name !== "" ? ( record.name + "<br />" ) : "";
```

Comment:
Shouldn't `record.name` be HTML escaped as well?

Reply:
It should be. I overlooked that detail.